### PR TITLE
fix-pd-recover

### PR DESCRIPTION
**Pull request by yiduoyunQ**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

When using alive PD to recreate pd cluster, there is no need to restart TiDB and TiKV, see TiUP related doc https://docs.pingcap.com/zh/tidb/stable/pd-recover#%E7%AC%AC-5-%E6%AD%A5%E6%89%A9%E5%AE%B9-pd-%E5%B9%B6%E5%90%AF%E5%8A%A8%E9%9B%86%E7%BE%A4

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by yiduoyunQ** [comment]
hi @KanShiori   please take a look, thanks.

**Comment by yiduoyunQ** [comment]
hi @ran-huang  please take a look when you have time, thanks.

File: en/pd-recover.md [hunk 1]
```diff
@@ -371,3 +364,5 @@ Use the following commands to restart the TiDB and TiKV clusters:
 kubectl delete pod -l app.kubernetes.io/component=tidb,app.kubernetes.io/instance=${cluster_name} -n ${namespace} &&
 kubectl delete pod -l app.kubernetes.io/component=tikv,app.kubernetes.io/instance=${cluster_name} -n ${namespace}
 ```
+
+Now TiDB cluster is recovered.
```
Comment (ran-huang) [inline en/pd-recover.md:368]:
```suggestion
Now the TiDB cluster is recovered.
```

File: en/pd-recover.md [hunk 2]
```diff
@@ -202,14 +202,7 @@ Check configuration. The following command uses placement rules as an example:
...
 
-### Step 6. Restart TiDB and TiKV
-
-Use the following commands to restart the TiDB and TiKV clusters:
-
-```shell
-kubectl delete pod -l app.kubernetes.io/component=tidb,app.kubernetes.io/instance=${cluster_name} -n ${namespace} &&
-kubectl delete pod -l app.kubernetes.io/component=tikv,app.kubernetes.io/instance=${cluster_name} -n ${namespace}
-```
+Now TiDB cluster is recovered.
```
Comment (ran-huang) [inline en/pd-recover.md:205]:
```suggestion
Now the TiDB cluster is recovered.
```

**Review by ran-huang — APPROVED** [review]

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**